### PR TITLE
All the changes we discussed over slack. 

### PR DESCRIPTION
**Pull request by abeebe-csumb**

### CODE REVIEW

**Review by erickakoyama — DISMISSED** [review]
lgtm 💯

File: sql/Project1SQL.sql [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    FOREIGN KEY (`preferred_pharmacy_id`)
+    REFERENCES `drug_db`.`pharmacy` (`pharmacy_id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = latin1;
+
+
+-- -----------------------------------------------------
+-- Table `drug_db`.`prescriptions`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `drug_db`.`prescriptions` (
```
Comment (erickakoyama) [inline sql/Project1SQL.sql:172]: Can we add a pharmacy_id column to prescriptions?

**Review by erickakoyama — CHANGES_REQUESTED** [review]
Sorry 🙏 , one thing below about prescriptions, then we should be good.

File: sql/Project1SQL.sql (hunk 1 above)
Comment (abeebe-csumb) [reply sql/Project1SQL.sql:172]: done

**Review by erickakoyama — APPROVED** [review]
lgtm

**Pull request merged**